docs/UserGuide.md

```diff
@@ -15,30 +22,35 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
+1. Ensure you have **Java 17 or above** installed in your Computer.
+   *  How To Install Java ([Windows](https://www.oracle.com/sg/java/technologies/downloads/#jdk23-windows) | [MacOS](https://www.oracle.com/sg/java/technologies/downloads/#jdk23-mac) | [Linux](https://www.oracle.com/sg/java/technologies/downloads/#jdk23-linux)
```

Comment:
These links doesn't show how to install Java, it only directs users to the downloads page for Java 23. Perhaps you could use the following links:

- Windows: https://se-education.org/guides/tutorials/javaInstallationWindows.html
- MacOS: https://se-education.org/guides/tutorials/javaInstallationMac.html
- Linux: https://se-education.org/guides/tutorials/javaInstallationLinux.html